xDrip-master.iml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file from the PR, it is local only. I thought there was .gitignore for this already. Thanks

Reply:
Sorry, being a noop in this. Shall i delete both files completely?